ui/timeline/src/commonMain/kotlin/social/androiddev/timeline/TimelineCard.kt

```diff
@@ -10,14 +10,7 @@
 package social.androiddev.timeline
 
 // import androidx.compose.desktop.ui.tooling.preview.Preview
-import androidx.compose.foundation.layout.Box
-import androidx.compose.foundation.layout.PaddingValues
-import androidx.compose.foundation.layout.Row
-import androidx.compose.foundation.layout.fillMaxWidth
-import androidx.compose.foundation.layout.padding
-import androidx.compose.foundation.layout.size
-import androidx.compose.foundation.layout.wrapContentHeight
-import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.foundation.layout.*
```

Comment:
Can we avoid wildcard imports?
https://sergiomartins8.medium.com/avoid-using-auto-asterisk-imports-on-intellij-idea-278b9a6f148c

Reply:
+1